cmd/experimental/kjobctl/pkg/cmd/create/create_test.go

```diff
@@ -699,7 +699,7 @@ func TestCreateCmd(t *testing.T) {
 			beforeTest: beforeSlurmTest,
 			afterTest:  afterSlurmTest,
 			args: func(tc *createCmdTestCase) []string {
-				return []string{"slurm", "--profile", "profile", "--", tc.tempFile}
+				return []string{"slurm", "--profile", "profile", "--", "--cpus-per-task", "2", tc.tempFile}
```

Comment:
Is this flag required? This test is more useful for Slurm with minimal flags.

Reply:
I added it in there to take advantage of this test case and avoid duplicating the full init-script but I can add a new test specifically for this flag.

Reply:
Moved flag to a different test case and fix calculations when n > 1.